package.json

```diff
@@ -10,12 +10,14 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "aws-sdk": "^2.1167.0",
```

Comment:
use `@aws-sdk/client-s3` sdk instead of `aws-sdk` as we only need the s3 functions

Reply:
lets do this later